packages/aws_signature_v4/test/s3/multiple_chunks_test.dart

```diff
@@ -0,0 +1,25 @@
+// Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import 'package:test/scaffolding.dart';
+
+import 'testdata/multiple_chunks_testdata.dart';
+
+// From: https://docs.aws.amazon.com/AmazonS3/latest/API/sigv4-streaming.html
```

Comment:
What is this in reference to? Ditto single chunk test

Reply:
This is where the test cases come from

Reply:
Hm I didn't see anything that looked like tests on that page - what am I missing? 😓

Reply:
It's towards the bottom of the page: https://docs.aws.amazon.com/AmazonS3/latest/API/sigv4-streaming.html#example-signature-calculations-streaming